src/Contract/Utxos.purs

```diff
@@ -21,14 +22,17 @@ import Plutus.FromPlutusType (fromPlutusType)
 -- | Results may vary depending on `Wallet` type. See `QueryM` for more details
 -- | on wallet variance.
 utxosAt
-  :: forall (r :: Row Type). Address -> Contract r (Maybe Transaction.UtxoM)
-utxosAt address = do
+  :: forall (r :: Row Type)
+   . Address
+  -> NetworkId
+  -> Contract r (Maybe Transaction.UtxoM)
+utxosAt address networkId = do
   plutusAddr <- liftContractM "utxosAt: unable to serialize address"
```

Comment:
any chance you could change this variable to `cardanoAddr` or `cslAddr` as mentioned in the issue? It's my fault it was like this haha.